documentation/projects/proposals/dark_mode/20240313-project_proposal_dark_mode.md

```diff
@@ -0,0 +1,81 @@
+
+**Author**: @fcoveram
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @obulat
+- [ ] @zackkrida
+
+## Project summary
+
+<!-- A brief one or two sentence summary of the project's features -->
+
+Include a feature that changes the UI from light to a dark color scheme.
+
+### Motivation
+
+Dark mode is a customization level that is in the spirit of adapting the UI to
+people’s contexts. The benefits span a11y, personal preferences set on the
+devices, energy efficiency in devices, and an alternative browsing experience of
+visual content due to the background contrast.
+
+## Goals
+
+<!-- Which yearly goal does this project advance? -->
+
+Refine Search Experience
+
+## Requirements
+
+<!-- Detailed descriptions of the features required for the project. Include user stories if you feel they'd be helpful, but focus on describing a specification for how the feature would work with an eye towards edge cases. -->
+
+1. Design documentation of pages and UI components in the Design Library and any
+   other additional Figma files.
+2. Design documentation of transition from current to new frontend
+   implementation.
+
+## Success
+
+<!-- How do we measure the success of the project? How do we know our ideas worked? -->
+
+- Add the dark theme to the site by allowing visitors to change the UI through
+  three options: Light theme, Dark theme, or follow device setting.
+- Document the dark variant of every UI element in both the Design Library and
+  Storybook.
+
+## Participants and stakeholders
+
+<!-- Who is working on the project and who are the external stakeholders, if any? Consider the lead, implementers, designers, and other stakeholders who have a say in how the project goes. -->
+
+- Lead: @fcoveram
+- Design: @fcoveram
+- Implementation: TBD
+
+## Infrastructure
+
+<!-- What infrastructural considerations need to be made for this project? If there are none, say so explicitly rather than deleting the section. -->
+
+TBD
+
+## Accessibility
+
+<!-- Are there specific accessibility concerns relevant to this project? Do you expect new UI elements that would need particular care to ensure they're implemented in an accessible way? Consider also low-spec device and slow internet accessibility, if relevant. -->
+
+- Follow the advisor of WCAG 2.2 to meet AA. Put special attention to the
+  [Perceivable section](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_overview&levels=aaa&showtechniques=321#principle1).
+
+## Marketing
+
+<!-- Are there potential marketing opportunities that we'd need to coordinate with the community to accomplish? If there are none, say so explicitly rather than deleting the section. -->
+
+Since the project involves visual changes across the site, there is a marketing
+opportunity to introduce the benefits of the UI’s dark theme.
```

Comment:
It would be good to think about this as much as possible now, rather than later. How much would we want to market this? The WordPress account would share socials for us, I am sure. Would we just want to create a screenshot of dark mode? Or a post on the make site showing a more detailed breakdown? Or a little UI tour video? What do you think? As the project lead, you're able to make these choices as to what you think is the best balance of effort vs. outcome.

Reply:
I like the idea of a short video showing the feature and quick glances of the three search results and the media details page, but sharing this point with [marketing team](https://wordpress.slack.com/archives/C0GKJ7TFA) for more ideas and tap into existing approaches I might not be aware of.